src/Request.hs

```diff
@@ -0,0 +1,63 @@
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE DataKinds #-}
+
+module Request (done, view, add, delete, reset) where
+-- TODO: maybe use qualified import?
+import Network.HTTP.Req
+    ( responseStatusCode,
+      port,
+      ignoreResponse,
+      req,
+      useHttpURI,
+      defaultHttpConfig,
+      runReq,
+      ReqBodyJson(..),
+      HttpMethod, jsonResponse, responseBody, HttpBodyAllowed, AllowsBody, CanHaveBody (CanHaveBody))
+import Text.URI (mkURI)
+import Data.Maybe (fromJust)
+import Data.Text (concat, pack)
+import Prelude hiding (concat)
+import Data.Aeson (Value, ToJSON)
+import Data.Kind (Type)
+import Control.Monad.IO.Class (MonadIO)
+import GHC.IO (liftIO)
+import Relude (lookupEnv, toText, fromMaybe, readMaybe)
```

Comment:
I thought you were not going to use relude. But if you are going to use relude, I suggest configuring it as a mixin, so it need not be explicitly imported.

Reply:
I am going to remove relude dependency as well, needed it for lookupEnv mainly. Once I move the lookupEnv to main I can use the one exported by System.Environment that returns an `IO (Maybe String)`

Reply:
I should add a TODO there